EIPS/eip-token-standard.md

```diff
@@ -0,0 +1,129 @@
+This is the suggested template for new EIPs.
+
+Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
+
+## Preamble
+
+    EIP: <to be assigned> (I Suggest 20)
+    Title: <EIP title>
+    Author: Fabian Vogelsteller <fabian@ethereum.org>, Vitalik Buterin <vitalik.buterin@ethereum.org>
+    Type: Informational
+    Category ERC
+    Status: Draft
+    Created: 2015-11-19
+
+
+## Simple Summary
+
+Token standard interface.
+
+
+## Abstract
+
+The following standard allows for people to implement a token standard API withing their smart contracts.
+
+This standard provides basic functionality for sending and approving tokens to be spend by a third party.
+
+
+## Motivation
+
+Following the same standard interface allows those tokens to be used in many wallets and exchanges.
```

Comment:
Another potential revision:

"A standard interface allows any tokens on Ethereum to be re-used by other applications: from wallets to decentralized exchanges."